readme.md

```diff
@@ -89,12 +89,17 @@ We can add to the context for a specific request as well:
 
 ```js
 let calledOnce = false;
-server.ext('onPostHandler', (request, reply) => {
-  if (!calledOnce) {
-    calledOnce = true;
-    server.plugins['hapi-view-context'].addContext(request, { someVariable: 'a totally different value' });
+server.route({
+  path: '/',
+  method: 'POST',
+  handler: (request, reply) => {
+    if (!calledOnce) {
+      calledOnce = true;
+      // addContext will only add to the context for the current request:
+      server.plugins['hapi-view-context'].addContext(request, { someVariable: 'a totally different value' });
```

Comment:
the idea of the plugin is to be able to add context to views outside of routes. Context can be added inside of routes using `reply.view('view', context)`